clam*: add Thai translation

nontw opened the pull request:
<!--
Thank you for contributing!
Please fill in the following checklist, removing items that do not apply.
See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
-->

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

navarroaxel commented on pages.th/linux/clamav.md:
```diff
@@ -0,0 +1,21 @@
+# clamav
```
Oh, the English page is not valid. This should be an alias page, or similar to https://github.com/tldr-pages/tldr/blob/main/pages/common/transmission.md

nontw replied:
Oh ok .. I just saw that too.  I just changed the original English content of clamav to be an alias of clamscan and adjusted the Thai page of clamav accordingly.

navarroaxel replied:
:thinking: Instead of the edition of the `clamav` page, translate `clamscan`. Let's work on `clamav` in another PR.

nontw replied:
Certainly.  This is done.  I have also removed clamav's Thai translation.

kbdharun commented on pages.th/common/clamdscan.md:
```diff
@@ -0,0 +1,36 @@
+
+`clamdscan {{path/to/file_or_directory}}`
+
+- แสกนข้อมูลจากสตรีมอินพุทมาตรฐาน
```
```suggestion
- แสกนข้อมูลจากสตรีมอินพุทมาตรฐาน:
```

kbdharun commented on pages.th/common/clamdscan.md:
```diff
@@ -0,0 +1,36 @@
+
+`clamdscan --infected`
+
+- ส่งผลการแสกนไปยังไฟล์ที่ระบุ
```
```suggestion
- ส่งผลการแสกนไปยังไฟล์ที่ระบุ:
```

kbdharun commented on pages.th/common/clamdscan.md:
```diff
@@ -0,0 +1,36 @@
+
+- แสกนข้อบกพร่องของไฟล์หรือไฟล์ในไดเรคทอรี
+
+`clamdscan {{path/to/file_or_directory}}`
```
Translate this path, and other command arguments inside `{{}}` throughout the pages.

nontw replied:
Done.

kbdharun commented on pages.th/common/clamdscan.md:
```diff
@@ -0,0 +1,36 @@
+
+`clamdscan --remove`
+
+- ใช้มัลติเธรดในการตรวจไดเรคทอรี
```
```suggestion
- ใช้มัลติเธรดในการตรวจไดเรคทอรี:
```

kbdharun commented on pages.th/common/clamdscan.md:
```diff
@@ -0,0 +1,36 @@
+
+`clamdscan --multiscan`
+
+- ส่งตัวอธิบายไฟล์ไปยังโปรแกรมแสกนพื้นหลังแทนการส่งไฟล์ข้อมูล
```
```suggestion
- ส่งตัวอธิบายไฟล์ไปยังโปรแกรมแสกนพื้นหลังแทนการส่งไฟล์ข้อมูล:
```

nontw commented:
Incorporate all the suggestions.  Didn't accidentally remove `:` this time :-)

navarroaxel commented on pages.th/common/clamdscan.md:
```diff
@@ -0,0 +1,36 @@
+
+- ส่งผลการแสกนไปยังไฟล์ที่ระบุ:
+
+`clamdscan --log {{path/to/log_file}}`
```
pla
Please translate the placeholders, like `path/to/quarantine_directory`, `path/to/log_file`, etc., for get this easier for non-English speakers.

nontw replied:
done

navarroaxel commented:
@nontw we have fixed the ClamAV pages in English version: 
- https://github.com/tldr-pages/tldr/blob/main/pages/common/clamav.md
- https://github.com/tldr-pages/tldr/blob/main/pages/common/clamdscan.md
- https://github.com/tldr-pages/tldr/blob/main/pages/common/clamscan.md
- https://github.com/tldr-pages/tldr/blob/main/pages/common/freshclam.md

Feel free to update this PR to include the full ClamAV translation in Thai

nontw commented:
> @nontw we have fixed the ClamAV pages in English version:
> 
> * https://github.com/tldr-pages/tldr/blob/main/pages/common/clamav.md
> * https://github.com/tldr-pages/tldr/blob/main/pages/common/clamdscan.md
> * https://github.com/tldr-pages/tldr/blob/main/pages/common/clamscan.md
> * https://github.com/tldr-pages/tldr/blob/main/pages/common/freshclam.md
> 
> Feel free to update this PR to include the full ClamAV translation in Thai

Yep, I added the translation for both Clamav and FreshClam.  Let me know if they're ok.

kbdharun reviewed (approved):
LGTM, Thanks for your contribution.
